rmgpy/data/kinetics/family.py

```diff
@@ -2758,63 +2758,6 @@ def add_atom_labels_for_reaction(self, reaction, output_with_resonance=True):
             for species in reaction.reactants + reaction.products:
                 species.generate_resonance_structures()
 
-    def get_w0(self, rxn):
```

Comment:
These functions are also listed under rmgpy.data.kinetics.family in rmg2to3.py. I am not sure if this needs to be updated, as what you really want is for these functions to instead call Arrhenius. But I doubt this will come up anyways

Reply:
I don't think the function has existed long enough for anyone to have scripts using it (except Matt). I added a change to remove them from rmg2to3.py.